docs/core/tools/dotnet.md

```diff
@@ -314,81 +223,57 @@ For more information about each tool, type `dotnet <tool-name> --help`.
 
-Specifies the location of the servicing index to use by the shared host when loading the runtime.
+- `DOTNET_ROLL_FORWARD` **Available starting with .NET Core 3.x SDK.**
 
-`DOTNET_CLI_TELEMETRY_OPTOUT`
+  Determines roll forward behavior. For more information, see the `--roll-forward` option earlier in this article.
 
-Specifies whether data about the .NET Core tools usage is collected and sent to Microsoft. Set to `true` to opt-out of the telemetry feature (values `true`, `1`, or `yes` accepted). Otherwise, set to `false` to opt into the telemetry features (values `false`, `0`, or `no` accepted). If not set, the default is `false` and the telemetry feature is active.
+- `DOTNET_ROLL_FORWARD_ON_NO_CANDIDATE_FX` **Available in .NET Core 2.x SDK.**
 
-`DOTNET_MULTILEVEL_LOOKUP`
+  Disables minor version roll forward, if set to `0`. For more information, see [Roll forward](../whats-new/dotnet-core-2-1.md#roll-forward).
 
-Specifies whether .NET Core runtime, shared framework, or SDK are resolved from the global location. If not set, it defaults to `true`. Set to `false` to not resolve from the global location and have isolated .NET Core installations (values `0` or `false` are accepted). For more information about multi-level lookup, see [Multi-level SharedFX Lookup](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/multilevel-sharedfx-lookup.md).
+- `DOTNET_CLI_UI_LANGUAGE`
 
-# [.NET Core 1.x](#tab/netcore1x)
-
-`DOTNET_PACKAGES`
-
-The primary package cache. If not set, it defaults to `$HOME/.nuget/packages` on Unix or `%userprofile%\.nuget\packages` on Windows.
-
-`DOTNET_SERVICING`
-
-Specifies the location of the servicing index to use by the shared host when loading the runtime.
-
-`DOTNET_CLI_TELEMETRY_OPTOUT`
-
-Specifies whether data about the .NET Core tools usage is collected and sent to Microsoft. Set to `true` to opt-out of the telemetry feature (values `true`, `1`, or `yes` accepted). Otherwise, set to `false` to opt into the telemetry features (values `false`, `0`, or `no` accepted). If not set, the default is `false` and the telemetry feature is active.
-
----
+  Sets the language of the CLI UI using a locale value such as `en-us`. The supported values are the same as for Visual Studio. For more information, see the section on changing the installer language in the [Visual Studio installation documentation](https://docs.microsoft.com/visualstudio/install/install-visual-studio?view=vs-2019). The .NET resource manager rules apply, so you don't have to pick an exact match&mdash;you can also pick descendants in the `CultureInfo` tree. For example, if you set it to `fr-CA`, the CLI will find and use the `fr` translations. If you set it to a language that is not supported, the CLI falls back to English.
 
```

Comment:
There are other env. variables supported by `dotnet.exe` (and the generated executables). Not sure if we should include them all, but for completeness:
* `DOTNET_DISABLE_GUI_ERRORS` - for GUI enabled generated executables - disables dialog popup which normally shows for certain class of errors (only writes to stderr and exits in that case).
* `DOTNET_ADDITIONAL_DEPS` - basically env. variable version of `--additional-deps`.
* `DOTNET_RUNTIME_ID` - override the detected RID
* `DOTNET_SHARED_STORE` - location of the "shared store" where assembly resolution falls back to in some cases
* `DOTNET_STARTUP_HOOKS` - list of assemblies to load and execute startup hooks from
* `COREHOST_TRACE`, `COREHOST_TRACEFILE`, `COREHOST_TRACE_VERBOSITY` - controls diagnostics tracing from the hosting components (`dotnet.exe`, `hostfxr`, `hostpolicy`, ...).